buildlib/RDMA_LinuxHeaders.cmake

```diff
@@ -95,4 +95,4 @@ rdma_check_kheader("rdma/ib_user_mad.h" "${DEFAULT_TEST}" NO_SHIM OPTIONAL)
 rdma_check_kheader("rdma/rdma_netlink.h" "int main(int argc,const char *argv[]) { return RDMA_NL_IWPM_REMOTE_INFO && RDMA_NL_IWCM; }")
 rdma_check_kheader("rdma/rdma_user_cm.h" "${DEFAULT_TEST}" NO_SHIM OPTIONAL)
 rdma_check_kheader("rdma/rdma_user_rxe.h" "${DEFAULT_TEST}")
-rdma_check_kheader("rdma/vmw_pvrdma-abi.h" "${DEFAULT_TEST}")
+rdma_check_kheader("rdma/vmw_pvrdma-abi.h" "int main(int argc,const char *argv[]) { return PVRDMA_UAR_SRQ_OFFSET; }")
```

Comment:
Move this diff hunk to the prior patch.

Reply:
Okay, the change has been moved to the first commit of the series.